sdk/storage/azure-storage-blob/conftest.py

```diff
@@ -12,5 +12,5 @@
 
 # Ignore async tests for Python < 3.5
 collect_ignore_glob = []
-if sys.version_info < (3, 5) or platform.python_implementation() == "PyPy":
-    collect_ignore_glob.append("tests/*_async.py")
+#if sys.version_info < (3, 5) or platform.python_implementation() == "PyPy":
+collect_ignore_glob.append("tests/*_async.py")
```

Comment:
Doesn't sound right :o

Reply:
Just temporary while I work on CI :)

Reply:
temporarily blocked this. pushing a commit with fixed async test and enabling again